letsencrypt-apache/letsencrypt_apache/display_ops.py

```diff
@@ -83,7 +83,8 @@ def _vhost_menu(domain, vhosts):
         code, tag = zope.component.getUtility(interfaces.IDisplay).menu(
             "We were unable to find a vhost with a ServerName "
             "or Address of {0}.{1}Which virtual host would you "
-            "like to choose?".format(domain, os.linesep),
+            "like to choose?\n(note: conf files with multiple "
```

Comment:
Rather than including this case in the select vhost / missing flag message, I think we should raise a separate error.  There isn't a flag the user can supply that will solve things here, right?